lib/galaxy/tools/deps/container_resolvers/mulled.py

```diff
@@ -54,8 +54,8 @@ def _package_hash(target):
 
 def list_docker_cached_mulled_images(namespace=None, hash_func="v2"):
     command = build_docker_images_command(truncate=True, sudo=False)
-    images_and_versions = subprocess.check_output(command).strip().split('\n')
-    images_and_versions = [line.split()[0:2] for line in images_and_versions[1:]]
+    images_and_versions = subprocess.check_output(command,shell=True).strip().split('\n')
```

Comment:
@ieguinoa we should never use shell=True if possible, which limitation do you try to solve here?

Reply:
It's just that the subprocess throws **OSError: [Errno 2] No such file or directory** if called with several parameters in a string and not a list. Shouldn't it be kept as a list when built and not parsed to string [here](https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tools/deps/docker_util.py#L196)